_includes/hreflangs.html

```diff
@@ -1,33 +1,6 @@
-<link rel="alternate" hreflang="fr" href="https://rms-support-letter.github.io/index-fr.html" />
-<link rel="alternate" hreflang="gl" href="https://rms-support-letter.github.io/index-gal.html" />
-<link rel="alternate" hreflang="el" href="https://rms-support-letter.github.io/index-gr.html" />
-<link rel="alternate" hreflang="hr" href="https://rms-support-letter.github.io/index-hr.html" />
-<link rel="alternate" hreflang="hy" href="https://rms-support-letter.github.io/index-hy.html" />
-<link rel="alternate" hreflang="id" href="https://rms-support-letter.github.io/index-id.html" />
-<link rel="alternate" hreflang="it" href="https://rms-support-letter.github.io/index-it.html" />
-<link rel="alternate" hreflang="ja" href="https://rms-support-letter.github.io/index-jp.html" />
-<link rel="alternate" hreflang="ko" href="https://rms-support-letter.github.io/index-ko.html" />
-<link rel="alternate" hreflang="lv" href="https://rms-support-letter.github.io/index-lv.html" />
-<link rel="alternate" hreflang="nl" href="https://rms-support-letter.github.io/index-nl.html" />
-<link rel="alternate" hreflang="pl" href="https://rms-support-letter.github.io/index-pl.html" />
-<link rel="alternate" hreflang="pt-BR" href="https://rms-support-letter.github.io/index-pt-br.html" />
-<link rel="alternate" hreflang="pt-PT" href="https://rms-support-letter.github.io/index-pt-pt.html" />
-<link rel="alternate" hreflang="ro" href="https://rms-support-letter.github.io/index-ro.html" />
-<link rel="alternate" hreflang="sr" href="https://rms-support-letter.github.io/index-rs.html" />
-<link rel="alternate" hreflang="ru" href="https://rms-support-letter.github.io/index-ru.html" />
-<link rel="alternate" hreflang="sv" href="https://rms-support-letter.github.io/index-se.html" />
-<link rel="alternate" hreflang="ta" href="https://rms-support-letter.github.io/index-ta.html" />
-<link rel="alternate" hreflang="tl" href="https://rms-support-letter.github.io/index-tl.html" />
-<link rel="alternate" hreflang="tr" href="https://rms-support-letter.github.io/index-tr.html" />
-<link rel="alternate" hreflang="uk" href="https://rms-support-letter.github.io/index-ua.html" />
-<link rel="alternate" hreflang="vi" href="https://rms-support-letter.github.io/index-vi.html" />
-<link rel="alternate" hreflang="zh-Hans" href="https://rms-support-letter.github.io/index-zh-cn.html" />
-<link rel="alternate" hreflang="zh-Hant" href="https://rms-support-letter.github.io/index-zh-tw.html" />
-<link rel="alternate" hreflang="en" href="https://rms-support-letter.github.io/" />
-<link rel="alternate" hreflang="x-default" href="https://rms-support-letter.github.io/" />
+{% for translation in site.translations %}
+  {%- assign hreflang = translation.url | replace: '/index-', '' | replace: '.html', '' -%}
+  <link rel="alternate" hreflang="{{hreflang}}" href="https://rms-support-letter.github.io{{translation.url}}" />
```

Comment:
IIRC `href` doesn't have to be absolute, so we don't need the https://rms-support-letter.github.io prefix

Reply:
I am referring to [this URL](https://developers.google.com/search/docs/advanced/crawling/localized-versions#all-method-guidelines), which contains the following information.

> Alternate URLs must be fully-qualified, including the transport method (http/https), so:
> https://example.com/foo, not //example.com/foo or /foo

Reply:
Oh. It's probably better to use `{{ translation.url | absolute_url }}` then though.

Reply:
Thanks! I fixed it with 7026475.